internal/buffer/save.go

```diff
@@ -113,7 +114,17 @@ func openFile(name string, withSudo bool) (wrappedFile, error) {
 		}
 	}
 
-	return wrappedFile{writeCloser, withSudo, screenb, cmd, sigChan}, nil
+	return wrappedFile{name, writeCloser, withSudo, screenb, cmd, sigChan}, nil
+}
+
+func (wf wrappedFile) Truncate(size int64) error {
```

Comment:
The `size` argument not used anymore? (It was not really needed in the first place, but now it is not even used.)